string/string.mbt

```diff
@@ -11,3 +11,184 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+pub fn iter(self : String, f : (Char) -> Unit) {
```

Comment:
it would be nice to split contributions to string in a separate package, so that review is a bit easier